src/python/test/test_dxclient.py

```diff
@@ -5750,6 +5750,15 @@ def get_name_and_policies(org_id=None):
         self.assertEqual(new_org_name, cur_org_name)
         self.assertEqual(new_org_policies, exp_org_policies)
 
+        # --saml-idp
+        proposed_idp = "samlprovider"
+        exp_org_policies = dict(cur_org_policies, restrictProjectTransfer=proposed_idp)
```

Comment:
It looks like we don't need `exp_org_policies` for this particular test.

Reply:
Thanks! Removed.